webapp/views.py

```diff
@@ -736,10 +736,17 @@ def dispatch_request(self, slug):
         if "article" not in context:
             return flask.abort(404)
 
-        # Redirect canonical annoucements
+        # Capture the original query string
+        original_query_string = flask.request.query_string.decode("utf-8")
```

Comment:
if this variable is used only inside the `if` block below, can you declare it there?